### PR TITLE
GH-24

### PR DESCRIPTION
**Pull request by akila-sama**

### CODE REVIEW

File: templates/retrieve-portfolio-posts.php
Comment (dharmendra-vadher) [inline templates/retrieve-portfolio-posts.php:1]: Write file comment. refer the documentation for detail.

**Comment by dharmendra-vadher** [comment]
- [x] change the following details of the plugin:
image

- [x] Add a button next to Deactivate on wp-admin/plugins.php page to go to your plugin page.
- [x] CSS and JS file should be loaded only on the page where shortcode is used.
- [x] Make the custom button like other links. change label from 'custom button' to something meaningful like plugin details or posts.
image
- [x] CSS and JS files should be loaded only on the plugin page on admin side.

File: akila-portfolio.php [hunk 1]
```diff
@@ -405,15 +420,15 @@ function ak_get_portfolio_posts_callback() {
...
 
 			echo '</tr>';
 		}
 
 		echo '</table>';
 		wp_reset_postdata();
 	} else {
-		echo '<p>' . esc_html__( 'No portfolio posts found.', 'my-plugin-ajax' ) . '</p>';
+		echo '<p>' . esc_html__( 'No portfolio posts found.', 'akila-portfolio' ) . '</p>';
 	}
```
Comment (dharmendra-vadher) [inline akila-portfolio.php]: put the above html code in template file.
check all the code for html content and move it into template file.

File: akila-portfolio.php [hunk 2]
```diff
@@ -172,28 +169,43 @@ function ak_recent_posts_by_category_shortcode( $atts ) {
...
-		'my_plugin',
-		array(
-			'ajax_url' => admin_url( 'admin-ajax.php' ),
-		)
-	);
+	if ( ! is_admin() && ak_has_shortcode( 'portfolio_submission_form' ) ) {
+		wp_enqueue_script( 'akila-portfolio-js-1', plugin_dir_url( __FILE__ ) . 'js/akila-portfolio.js', array( 'jquery' ), '1.0', true );
+		wp_localize_script(
+			'akila-portfolio-js-1',
+			'my_plugin',
```
Comment (dharmendra-vadher) [inline akila-portfolio.php]: User prefix for my_plugin

File: akila-portfolio.php [hunk 3]
```diff
@@ -345,26 +357,29 @@ function ak_custom_portfolio_sortable_columns( $columns ) {
...
-		'my_plugin',
-		array(
-			'ajax_url' => admin_url( 'admin-ajax.php' ),
-		)
-	);
+	if ( is_admin() ) {
+		wp_enqueue_script( 'akila-portfolio-js', plugin_dir_url( __FILE__ ) . 'js/akila-portfolio.js', array( 'jquery' ), '1.0.0', true );
+		wp_localize_script(
+			'akila-portfolio-js',
+			'my_plugin',
```
Comment (dharmendra-vadher) [inline akila-portfolio.php]: use prefix

**Review by dharmendra-vadher — CHANGES_REQUESTED** [review]
